src/backend/distributed/deparser/deparse_view_stmts.c

```diff
@@ -17,6 +17,7 @@
 #include "distributed/commands.h"
 #include "distributed/deparser.h"
 #include "distributed/listutils.h"
+#include "distributed/log_utils.h"
```

Comment:
can you undo the changes in this file? That is mostly to keep the git history clean